core/core.md

```diff
@@ -130,3 +130,15 @@ As noted in Non-Goal #3 in the [goals doc](../dogma/goals.md), this is not somet
 As mentioned above, for an honest members to successfully connect to the rest of the honest community; they must not be eclipsed by dishonest Hosts. If an honest member has no choice but to access the community through a dishonest Host, then Fora can make no guarantee that the member can be protected from banned members and banned content.
 
 Thus the Fora protocol assumes that the peer-to-peer network is sufficiently distributed and decentralized such that no member can be fully eclipsed by dishonest Hosts.
+
+## Dishonest Members
+
+Members who consistently or flagrantly break community rules can be banned by Enforcers. However, these bad actors can simply create a new identity and try to enter the community again. Thus, Fora communities facing these Sybil attacks, should tighten their admission requirements to prevent banned users from re-entering the community with freshly minted identities.
+
+This can be achieved either by evaluating whether the proposed member is an established member in other Fora communites, or by relying on attestations from members who are already in the community.
+
+Since member identities must be registered on a public ledger (blockchain), this provides a measure of spam prevention since most public blockchains will impose an inherent cost for user registration through their transaction fees. These can be tuned to higher or lower spam prevention by increasing the cost associated with creating a new member, this can be done either Proof-Of-Stake style (by increasing the cost in tokens for creating a new user) or Proof-of-Work style (by requiring a unique solution to a cryptographic puzzle in order to create a user). Either approach imposes a cost to creating a new Fora identity, and thus incentivizes members to preserve their reputations. Note this reduces accessibility for users that cannot afford to pay such costs, thus Fora communities must strike a balance between spam prevention and accessibility. Communities should also find ways to admit members that cannot own high-value identities, perhaps by requiring more attestations to join a community if the user deposit is too low.
+
+Spammers and malicious users may still exist, but each banned identity will have its reputation ruined and such users will have to continually pay to create new identities. These new identities in turn will not be able to re-enter well-moderated communities without gaining some reputation by joining and behaving in other communities or getting vouched for by existing members.
```

Comment:
It might be non-trivial to lookup the history of a DID if the blockchain hosting it doesn't maintain historical info. Even with historical info, the admittor would need to look through all the DID doc changes and check to see if the account was banned from any communities OR they could query a list well known communities and cross check the ban-list. 

This will likely become a background check service, but it might be good to note that this isn't trivial and likely won't be a feature for a long time

Reply:
This is still pretty trivial imo since the bans are recorded on public blockchains and directly associated with the DID (either through the DID doc or DID metadata). I'm not relying on historical info at all. I see this being implemented in two ways:

1. Fora-specific information is written directly into DID doc. This is the easiest case, since the fact that the user had been banned from CommunityA will be written directly into the current DID doc by the enforcer of CommunityA. This isn't historical information. It is part of the current state of the DID doc that the subject was banned from these communities. Of course, this requires that the particular part of the DID doc that could store such information is only writable by the relevant enforcers of that community.

2. In the case where we don't store Fora-specific information in the DID doc (this is where im currently leaning), the information that this user has been banned will be stored as metadata on the public blockchain of that particular Fora community. This **still means** that the user's DID is publically and permanently (subject to unban) associated with a community ban. Of course, an admittor will have to know where to look for this metadata to find it, but it is public and therefore theoretically findable (and maybe instantly findable with future services). Fora should be implemented to make this information as easy to find as possible (The store key should be keyed under the DID first ie. `{did}/{community}/banned => true`, so admittors don't have to search through particular communities, i will explain in detail in the spec).
Fora admittors can then just search the DID for bans on the most popular Fora blockchains. They may also create a waiting period where other users can submit ban proofs that prove a user got banned from a particular community, if the new member did not volunteer the information already this is grounds for denying admission.

So either way, there are pretty sensible ways to get this information to admittors since all this info is public and directly associated with the DID in question.

Reply:
> Fora-specific information is written directly into DID doc. 

If you are referring to the user DID doc, I don't think this is a feasible solution. We can discuss it at length more though

2 Seems more likely. And while I agree it is "sensible" I still don't believe it is trivial. This requires a lot of infrastructure support, and this is something I don't think communities would be able to successfully do for quite sometime. My concern isn't that it isn't possible, but that a banned user might be able to join other communities without consequence in the first few years. I'd just like to make a distinction between what is natively supported and what requires external tooling